### DISCUSSION TITLE
Questions about the NS module

### DISCUSSION THREAD

**Discussion by Ethan-xj** [idaholab/moose#27089] [Q&A Modules: Navier-Stokes] [answered]
@GiudGiud Hello guys, I know that the Reactor module can create complex geometry meshes that are related to reactors. Can I use the NS module on the mesh which is created by the Reactor module?

Ethan

**Comment by GiudGiud** [comment, marked as answer]
Hello

Yes but you'll want to keep the number of elements down until we merge the scalable solver.

Guillaume

**Reply by lindsayad** [reply]
We have scalable solvers for moderate Reynolds number for finite elements
